src/TW/CodeGen/Flow.hs

```diff
@@ -0,0 +1,121 @@
+{-# LANGUAGE OverloadedStrings #-}
+module TW.CodeGen.Flow
+    ( makeFileName, makeModule
+    , libraryInfo
+    )
+where
+
+import Prelude
+
+import TW.Ast
+import TW.BuiltIn
+import TW.JsonRepr
+
+import Data.Monoid
+import System.FilePath
+import qualified Data.List as L
+import qualified Data.Text as T
+
+makeFileName :: ModuleName -> FilePath
+makeFileName (ModuleName parts) =
+    (L.foldl' (</>) "" $ map T.unpack parts) ++ ".js"
+
+libraryInfo :: LibraryInfo
+libraryInfo = LibraryInfo "Flow" "flow-typed-wire-utils" "0.1.0.0"
```

Comment:
is this an npm library? is it needed for the generated output and where can I find the source?